deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -158,13 +162,11 @@ public void runJob() {
 					+ ", types does not support children");
 		}
 		String groups = depositStatus.get(DepositField.permissionGroups.name());
-		AccessGroupSet groupSet = new AccessGroupSet(groups);
+		
 		// Verify that the depositor is allow to ingest to the given destination
-		if (!destObj.getAccessControls().hasPermission(groupSet, Permission.ingest)) {
-			failJob("Cannot add children to destination",
-					"Depositor does not have permissions to ingest to destination "
-							+ destObj.getPid().getRepositoryPath());
-		}
+		aclService.assertHasAccess(
```

Comment:
A small thing, but maybe we can store destObj.getPid() in a variable to avoid the duplicate calls